modules/machineset-yaml-rhv.adoc

```diff
@@ -0,0 +1,110 @@
+<16>// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-rhv.adoc
+
+[id="machineset-yaml-rhv_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on {rh-virtualization}
+
+This sample YAML defines a machine set that runs on
+{rh-virtualization} and creates nodes that are labeled with
+`node-role.kubernetes.io/<node_role>: ""`
+
+In this sample, `<infrastructure_id>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and `<role>` is the node label to add.
+
+[source,yaml,subs="+quotes"]
+----
+    apiVersion: machine.openshift.io/v1beta1
+    kind: MachineSet
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+        machine.openshift.io/cluster-api-machine-role: <role> <2>
+        machine.openshift.io/cluster-api-machine-type: <role> <2>
+      name: <infrastructure_id>-<role> <3>
+      namespace: openshift-machine-api
+    spec:
+      replicas: <number_of_replicas> <4>
+      Selector: <5>
+        matchLabels:
+          machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+          machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+      template:
+        metadata:
+          labels:
+            machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+            machine.openshift.io/cluster-api-machine-role: <role> <2>
+            machine.openshift.io/cluster-api-machine-type: <role> <2>
+            machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+        spec:
+          metadata:
+            labels:
+              node-role.kubernetes.io/<role>: "" <2>
+          providerSpec:
+            value:
+              apiVersion: ovirtproviderconfig.machine.openshift.io/v1beta1
+              cluster_id: <ovirt-cluster-id> <6>
```

Comment:
This replaceable value uses hyphens, and should instead use underscores. Please check the rest of them in this YAML, because there are a few others.

Reply:
Fixed.